<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorWidget</class>
 <widget class="QWidget" name="EditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="CodeEditor" name="codeEdit"/>
   </item>
   <item>
    <widget class="QWidget" name="findReplaceWidget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Find:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="findNextButton">
        <property name="text">
         <string>Find Next</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="searchField"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="replaceButton">
        <property name="text">
         <string>Replace</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="findPreviousButton">
        <property name="text">
         <string>Find Previous</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="replaceField"/>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="replaceFindButton">
        <property name="text">
         <string>Replace &amp;&amp; Find</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Replace With:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="replaceAllButton">
        <property name="text">
         <string>Replace All</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>editor/codeeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
